### DISCUSSION TITLE
I did a review of Ice that I think you'll enjoy

### DISCUSSION THREAD

**Discussion by podfeet** [jordanbaird/Ice#171] [General]
I've been a big fan of Bartender for ages but with the recent kerfuffle I knew people were interested in alternatives.  Hope you like the review - and I'll post the "wee bug" I uncovered under issues.

[www.podfeet.com/...](https://www.podfeet.com/blog/2024/06/ice-bartender-replacement/)

**Comment by jordanbaird** [comment]
Wow, thanks for this! That was a very thourough review.

The display bug is sadly one of the more common issues. I may have come up with a fix for it, but unfortunately, I can't test it myself as I don't have a MacBook with a notch (the bug doesn't occur without one).

As mentioned in your comments, there is a [beta](https://github.com/jordanbaird/Ice/issues/1#issuecomment-2176647022) available that has a secondary bar! Although, be prepared for some bugs.

On a final note, if you choose a custom image, you can actually have it display as a monochrome image, as if it were a default icon in the menu bar. You just have to check the option to "Use template image". Maybe it could use a better name.

<img width="358" alt="image" src="https://github.com/jordanbaird/Ice/assets/90936861/dda7190c-7ba9-49be-9fa1-600d6124bd07">

<img width="659" alt="image" src="https://github.com/jordanbaird/Ice/assets/90936861/9ee32738-7d94-47a6-80e4-3d763e176bb2">

Thanks again for the review!

**Comment by podfeet** [comment]
Glad you liked it! And if I can test the bug for you let me know. I'm very good at finding issues...

That's very cool you can do monochrome - I didn't catch that. Love that you took my little podfeet for your screenshot. :-)

I'm very excited about a secondary bar, but I may not get a chance to test out the beta for the next couple of weeks.  I have some ideas for your documentation request that I'll post over in the new issue you posted.

I assume it's ok that I quote what you said in my review as an addendum...

**Reply by jordanbaird** [reply]
> I assume it's ok that I quote what you said in my review as an addendum...

Of course!
